CONFIGURATION.md

```diff
@@ -123,7 +123,7 @@ dr_msg_cb                                |  P  |                 |
 
 Property                                 | C/P | Range           |       Default | Importance | Description              
 -----------------------------------------|-----|-----------------|--------------:|------------| --------------------------
-request.required.acks                    |  P  | -1 .. 1000      |             1 | high       | This field indicates how many acknowledgements the leader broker must receive from ISR brokers before responding to the request: *0*=Broker does not send any response/ack to client, *1*=Only the leader broker will need to ack the message, *-1* or *all*=broker will block until message is committed by all in sync replicas (ISRs) or broker's `min.insync.replicas` setting before sending response.  <br>*Type: integer*
+request.required.acks                    |  P  | -1 .. 1000      |            -1 | high       | This field indicates the number of acknowledgements the leader broker must receive from ISR brokers before responding to the request: *0*=Broker does not send any response/ack to client, *1*=Only the leader broker will need to ack the message, *-1* or *all*=broker will block until message is committed by all in sync replicas (ISRs) or broker's `min.insync.replicas` setting before sending response. <br>*Type: integer*
```

Comment:
>*-1* or *all*=broker will block until message is committed by all in sync replicas (ISRs) or broker's `min.insync.replicas` setting before sending response.

Is this correct? If there are 4 replicas in the ISR and `min.insync.replicas=2` 2 acks won't satisfy my request. So it's not really an `or` so much that `min.insync.replicas` sets the bare minimum number of in-sync replicas needed in order to deem a partition healthy enough to accept new messages from my producer. 

It might be helpful to break these two concepts up to make this more clear. 

> *-1* or *all*=broker will block until message is committed by all in sync replicas (ISRs). If there are less than  `min.insync.replicas` in the ISR the produce request will fail.